recipes/scotch/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "scotch" %}
+{% set version = "6.0.4" %}
+{% set md5 = "d58b825eb95e1db77efe8c6ff42d329f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: scotch_6.0.4.tar.gz
+  md5: {{ md5 }}
+  url: http://gforge.inria.fr/frs/download.php/file/34618/scotch_6.0.4.tar.gz
```

Comment:
I changed it, but because the number in the url changes for every file I initially didn't.